nbconvert/nbconvertapp.py

```diff
@@ -238,21 +238,20 @@ def initialize(self, argv=None):
 
 
     def init_syspath(self):
-        """
-        Add the cwd to the sys.path ($PYTHONPATH)
-        """
+        """Add the cwd to the sys.path ($PYTHONPATH)"""
```

Comment:
I understand that approach with executable code and, in general, I would leave style as it is within a document if I wasn't editing the document already. I'm happy to put it back as it was. Though, I could make the case that beautiful docstrings encourage more docstrings ;-)